README.md

```diff
@@ -1,5 +1,32 @@
 # Constitution
 
-The governing rules of Hackers at Cambridge.
+The governing rules of "Hackers at Cambridge", hereafter (and in the rest of the repository) referred to as the "Society".
 
-[Read the constitution here](constitution.md).
+[Read the constitution here](constitution.md). You can also find a web version
+[on our website](https://hackersatcambridge.com/constitution).
+
+# This Repository
+
+As laid out in the [constitution](constitution.md#society-repository), this repository is used to list all rules and guidelines of the Society that are not defined or contained in the constitution. This repository exists for the benefit of the executive committee, but is free to read for the general public for those who may be interested.
+
+As for why we went with this setup, we will be posting more information about this soon!
+
+# Our Website
+
+The official website of the Society is located at [https://hackersatcambridge.com](https://hackersatcambridge.com).
+
+# Making changes to this repository
+
+Any changes to this repository must require a majority vote at a meeting of the Executive Committee may issue a change to the Repository's documents. Please view [the template](.github/PULL_REQUEST_TEMPLATE.md) for more information. Individual files or folders may relax these requirements when changes are made to them, but these requirements are *only* relaxed for those changes which affect those files and folders.
+
+However, if changes to this repository affect the constitution document on this repository in *any* way, the requirements for changes to the constitution as outlined within it *must* be met. In cases where this repository contradicts the constitution, the constitution always takes precedence.
```

Comment:
This is confusing wording. How about "Any changes to the constitution document must themselves meet the constitutional requirements"